src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -209,6 +209,32 @@ class TinyMCEConfig extends HTMLEditorConfig
      */
     protected $contentCSS = null;
 
+
+    /**
+     * List of image size preset that will appear when you select an image. Each preset can have the following:
+     * * `name` to store an internal name for the preset (required)
+     * * `i18n` to store a translation key (e.g.: `SilverStripe\Forms\HTMLEditor\TinyMCEConfig.BESTFIT`)
+     * * `text` that will appear in the button (should be the default English translation)
+     * * `width` which will define the horizontal size of the preset. If not provided, the preset will match the
+     *   original size of the image.
+     * @var array
+     * @config
+     */
+    private static $image_size_presets = [
+        [
+            'width' => 600,
+            'i18n' => self::class . '.BEST_FIT',
+            'text' => 'Best fit',
+            'name' => 'bestfit',
+            'default' => true
+        ],
+        [
+            'i18n' => self::class . '.ORIGINAL',
+            'text' => 'Original',
+            'name' => 'originalsize'
+        ]
```

Comment:
I think the original size should always exist and should be enforced on the front-end side anyways? Shall we remove it from here?

Reply:
I'm not overly fuss either way. I suspect very few people will actually care enough to customise those options. The only reason I made all of this customisable is because it wasn't any more difficult than hard coding the two options in.

It would make it slightly more complex, because you would have a mix of options you can customise implemented one way and another one you can't customise implemented a different way.

Reply:
I reckon "original" shouldn't be considered by the backend at all, that could potentially simplify things on both sides - backend and frontend, and make the configs more explicit and straghtforward.

I don't feel too strong about this, so you decide.

Me reasoning is in the other thread: https://github.com/silverstripe/silverstripe-asset-admin/pull/1018#issuecomment-544703136